Detectors/FIT/FV0/simulation/src/Digitizer.cxx

```diff
@@ -100,14 +89,31 @@ void Digitizer::process(const std::vector<o2::fv0::Hit>& hits,
   LOG(INFO) << "[FV0] Digitizer::process(): begin with " << hits.size() << " hits";
 
   auto sorted_hits{hits};
-  std::sort(sorted_hits.begin(), sorted_hits.end(), [](o2::fv0::Hit const& a, o2::fv0::Hit const& b) {
-    return a.GetTrackID() < b.GetTrackID();
-  });
-
+  std::sort(std::begin(sorted_hits), std::end(sorted_hits),
+            [](o2::fv0::Hit const& a, o2::fv0::Hit const& b) {
+              return a.GetTrackID() < b.GetTrackID();
+            });
+
+  auto const PmtResponseVc = [&](Vc::float_v x) -> Vc::float_v {
+    if (x[0] > 2 * DigitizationParameters::mPmtTransitTime) {
+      return Vc::float_v(0);
+    }
+    if (x[Vc::float_v::Size - 1] < -DigitizationParameters::mPmtTransitTime) {
+      return Vc::float_v(0);
+    }
+    x += Vc::float_v(DigitizationParameters::mPmtTransitTime);
+    Vc::float_v const x2 = x * x;
+    Vc::float_v const c = Vc::float_v(-1.0f / DigitizationParameters::mPmtTransitTime2);
```

Comment:
This division is not necessary (since dividing by a constant). Replacing it with a multiplication will make the code faster and lead to better SIMD speedup.

Reply:
Done

Reply:
> This division is not necessary (since dividing by a constant). Replacing it with a multiplication will make the code faster and lead to better SIMD speedup.

I tried this before and saw now speedup, so probably the compiler computes the inversion at compile time (have not looked at the assembly code). In addition the runtime of this loop is dominated by memory in/out operations.

Reply:
Well. Better to be sure about it in particular in hotspot kernels. Divisions are usually not optimized out when not using fast-math.